frontend/src/admin/reducers/__tests__/userReducers-test.js

```diff
@@ -12,11 +12,32 @@ describe('user reducers', () => {
         type: 'USER_DETAILS_RECEIVED',
         payload: {
           type: 'SUPER',
+          email: 'someone@mail.com',
+          somethingElse: 'another thing',
         },
       });
       expect(reduction).toEqual('SUPER');
     });
   });
+
+  describe('email', () => {
+    it('defaults to an empty string', () => {
+      const reduction = type(undefined, { email: 'donald@trump.com' });
```

Comment:
Interesting choice of email address... 🤔

Reply:
: )) He was the person on my mind at the time, for obvious reasons... :'(